reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -49,6 +49,7 @@ import org.ossreviewtoolkit.model.licenses.filterExcluded
 import org.ossreviewtoolkit.model.utils.ResolutionProvider
 import org.ossreviewtoolkit.reporter.Reporter
```

Comment:
Commit message nit: I rather like full sentences in the title, i.e., "for licenses **that are** not present" .